frontend/include/prelude.html

```diff
@@ -13,5 +13,8 @@
 
 <meta name="theme-color" />
 <link rel="stylesheet" href="/frontend/global/global.css" />
-<script src="/frontend/global/helpers.js"></script>
+<script
+	src="https://polyfill.io/v3/polyfill.min.js?features=Array.prototype.every,Array.prototype.filter,Array.prototype.flat,Array.prototype.includes,Array.prototype.map,ArrayBuffer,document,fetch,JSON,localStorage,Map,Number.parseFloat,Promise,Set,String.prototype.includes,Symbol.iterator,URL,WeakMap,WeakSet"
```

Comment:
You've removed the `String.prototype.replaceAll` polyfill from **frontend/global/helpers.js**, is there a reason why you didn't add it back here? I don't see it included in the URL.  Unless they add it in by default I feel like this would bring that old issue back.

Reply:
I'll add it manually. I used a script to find all the required polyfills and I guess that one wasn't added.